doc/reference/causality.rst

```diff
@@ -0,0 +1,7 @@
+=========================================
+`causality` - Granger causality measures
```

Comment:
No ticks in the title. Just call it "Granger causality measures". It could be renamed to just "Causality measures" once you add other measures.

Reply:
Could you please elaborate. Do you mean something like this:
```
======================================
causality - Granger causality measures
```
Or do you want to get rid of "causality" altogether?

Reply:
> Or do you want to get rid of "causality" altogether?

yes. I suggest

```
======================
Granger causality measures
```

Reply:
Addressed in commit 74c933a.